app/grandchallenge/reader_studies/views.py

```diff
@@ -1134,7 +1134,11 @@ class AnswerViewSet(
     serializer_class = AnswerSerializer
     queryset = (
         Answer.objects.all()
-        .select_related("creator", "question__reader_study")
+        .select_related(
+            "creator",
+            "question__reader_study",
+            "display_set",
```

Comment:
I don't think adding display_set here is necessary, did you check in the debug toolbar?

Reply:
I queried Mike about if it would make sense to add it here. What would you expect to see in the debug toolbar? (Inexperience in Django optimalisations).

Reply:
Heh, just a single glance to remind me what `select_related` does already points myself to the correct check. If any DB queries are performed to fetch the actual display_set... they should be added.

Will check.

Reply:
Yes, that's it. You only need it if you need any information from the display sets in this view. If you're just serializing a hyperlink to the display set then that can already be generated without fetching the extra data as that is a foreign key already in the answer table. Always best to check by counting the number of SQL queries for the view with multiple rows.

Reply:
Doesn't seem to matter much. So I have removed the select_related. Good catch @jmsmkn .